pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
+
+- Execute a command (using the command module) on a group of hosts:
+
+`ansible {{group}} --module-name command --args '{{my command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} --become --ask-become-pass --module-name command --args '{{my command}}'`
+
+- Execute a command using a custom inventory file:
+
+`ansible {{group}} --inventory-file {{path/to/inventory/file}} --module-name command --args '{{my command}}'`
```

Comment:
I thought we had agreed that the [] syntax was superfluous when the short option letter is the first letter of the expanded word. Do you recall otherwise?